apple/Tests/TestCases/Prelude/SargonErrorTests.swift

```diff
@@ -23,4 +23,14 @@ final class SargonErrorTests: Test<SargonError> {
 		let sut = SUT.UnknownNetworkForId(badValue: 99)
 		XCTAssertEqual(sut.debugDescription, "10049: No network found with id: '99'")
 	}
+
+	func test_localized_description_for_sensitive_error() {
+		let sut = SUT.UnknownNetworkForId(badValue: 99)
+		XCTAssertEqual(sut.localizedDescription, "Error code: 10049")
+	}
+
+	func test_localized_description_for_non_sensitive_error() {
+		let sut = SUT.FailedToDeserializeJsonToValue(jsonByteCount: 100, typeName: "TypeName")
+		XCTAssertEqual(sut.localizedDescription, "Error code: 10070\nError message: Failed deserialize JSON with #100 bytes to value of type TypeName")
```

Comment:
This test will fail since the error message was updated:
```
"Failed deserialize JSON with #{json_byte_count} bytes to value of type {type_name} with error: \"{serde_message}\""
```

Reply:
Forgot to update after merge. Thanks